svc/krane/internal/deployment/state_test.go

```
@@ -0,0 +1,117 @@
+	}}
+
+	runningNoCondition := podBase("pod-no-condition")
+	runningNoCondition.Status.Phase = corev1.PodRunning
+
+	pending := podBase("pod-pending")
+	pending.Status.Phase = corev1.PodPending
+
+	failed := podBase("pod-failed")
+	failed.Status.Phase = corev1.PodFailed
+
+	client := fake.NewSimpleClientset(
```

golangci-lint (2.11.4)

[error] SA1019: fake.NewSimpleClientset is deprecated: NewClientset replaces this with support for field management, which significantly improves server side apply testing. NewClientset is only available when apply configurations are generated (e.g. via --with-applyconfig). 

(staticcheck)